tests/concepts/texplain.nim

```diff
@@ -1,6 +1,18 @@
 discard """
   cmd: "nim c --verbosity:0 --colors:off $file"
-  disabled: "true"
+  disabled: "windows"
```

Comment:
This should be a known issue, broke because of nkError changes.

Reply:
does nkError have a github issue?  i found [this one ]( https://github.com/nim-lang/Nim/pull/16915 ) , but is there a original?

Reply:
This is the project: https://github.com/nim-works/nimskull/projects/1#column-16654467

The done column has the pr that broke it. Looking at projects tab https://github.com/nim-works/nimskull/projects you can see a big ol' description of nkError if you scroll down.